### ISSUE TITLE
Integrated into Blender Screenwriter

### ISSUE THREAD

**Issue by tin2tin** [gabrielmontagne/blender-addon-unfurl-fountain#1]
I've used your script in the latest update of Blender Screenwriter in order to use your timings and text displays. I hope that's ok?

**Comment by gabrielmontagne** [comment]
That is super cool. Apologies, @tin2tin , I had promised to contribute this directly myself -- but I've been stuck on a couple of medium... personal problems : /   

I'm super happy you brought this in.  Thanks!

**Comment by tin2tin** [comment]
Oh, I hope you're okay. And thank you.

I've also added option to assign 3d objects to words in the screenplay, so when generating scenes, the 3d objects will be linked into the scenes where the word appear. Ex. character "boy" is talking in a scene, then the 3d object assigned to "boy" will be auto linked into that scene, and all other scenes he appear. 

I thinking about merging my scene strip tools into it so the newly generated scenes can preview scene strip camera cuts in the 3d view, but it might be nice to have the dialogue in the 3d view too. If I'm going to try to do this, would it be okay if I borrowed a bit of your text-strip to 3d-view text script?
